i18n/en.json

```diff
@@ -1953,15 +1953,14 @@
   "center_panel.direct.closeDirectMessage": "Close Direct Message",
   "center_panel.direct.closeGroupMessage": "Close Group Message",
   "change_url.close": "Close",
-  "change_url.endWithLetter": "URLs must end with a lowercase letter or number.",
-  "change_url.helpText": "You can use lowercase letters, numbers, dashes, and underscores.",
-  "change_url.invalidDirectMessage": "User IDs are not allowed in channel URLs.",
+  "change_url.endWithLetter": "URL must end with a letter or number.",
   "change_url.invalidUrl": "Invalid URL",
-  "change_url.longer": "URLs must have at least 2 characters.",
-  "change_url.startAndEndWithLetter": "URLs must start and end with a lowercase letter or number.",
-  "change_url.startWithLetter": "URLs must start with a lowercase letter or number.",
+  "change_url.longer": "URL must be two or more characters.",
+  "change_url.noUnderscore": "URL can not contain two underscores in a row.",
```

Comment:
Thanks, @jespino. This is very strange as I didn't make these changes. The only change I made for this PR is for "delete_channel_question". 

I wonder if these are from another, previous PR and got added to this one because I didn't merge/rebase?